src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Value/AttributeMatchCollection.php

```diff
@@ -69,4 +69,13 @@ public function offsetGet($offset)
     {
         return isset($this->matches[$offset]) ? $this->matches[$offset] : null;
     }
+
+    public function getAttributes()
```

Comment:
Please remove duplicate code. See remark above.

Reply:
Also, see my response to your feedback. What should be changed in this method is the call to `jsonEncode` on the `$match`

Reply:
We could address this later on as suggested.